### PR TITLE
Compare the files by gzip size

### PR DESCRIPTION
**Pull request by piuccio**
As an optional argument the size can be expressed as the gzip length of the minified files.

Once again this PR is rebased on top of #27 and #28. Let me know if you plan to merge any of those.
This would also solve #17 

Cheers

### CODE REVIEW

**Comment by jakubpawlowicz** [comment]
Rebasing may be a bit more tricky now but please do so.

**Comment by piuccio** [comment]
Thanks for reviewing, I'll rebase in the coming days once back to work

**Comment by jakubpawlowicz** [comment]
Thanks @piuccio! Great contributions so far :100:

**Comment by piuccio** [comment]
I've rebased :wink: 
The other two pull requests want to be rebased as well, so please don't review them until this one gets merged.

**Comment by jakubpawlowicz** [comment]
Sure :+1:

File: lib/measure.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  if (type === 'byte') {
+    return value + ' bytes';
+  }
+  if (type === 'time') {
+    return value + ' ms';
+  }
+};
+
+module.exports = Measure;
```
Comment (jakubpawlowicz) [inline lib/measure.js]: Very nice module :+1:

File: lib/minify.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+'csswring': function (source) {
+  return csswring.wring(source).css;
+},
+'more-css': function(source) {
+  return moreCss.compress(source, true);
+},
+'ncss': ncss,
+'sqwish': sqwish.minify,
+'ycssmin': ycssmin.cssmin
+};
```
Comment (jakubpawlowicz) [inline lib/minify.js]: You missed indents here, likely a copy and paste issue.

**Comment by jakubpawlowicz** [comment]
I've fixed the formatting issues and merged it manually.

**Pull request closed without merging**